app/controllers/api/membership_requests_controller.rb

```diff
@@ -7,6 +7,12 @@ def pending
     respond_with_collection
   end
 
+  def my_pending
```

Comment:
I'm not in love with these new pending methods; we should be using visible_records to define this.

```
def pending
  instantiate_collection { |collection| collection.pending }
  respond_with_resource
end

def previous
  instantiate_collection { |collection| collection.responded_to }
  respond_with_collection
end

private

def visible_records
  load_and_authorize :group
  @group.membership_requests
end
```